docs/ops.md

```diff
@@ -0,0 +1,215 @@
+  threads that should be spawned to handle tasks. If not provided will default
+  to the number of logical CPU cores available.
+
+### Logging
+
+Settings to control the format and amount of logs generated.
+
+- `logging.format` - The format to emit logs in. One of
+
+  - `pretty` (default in development) - Multiple lines per event, human-oriented
+    formatting and color.
+  - `compact`- A single line per event, with formatting and colors.
+  - `mozlog` (default in production) - A single line per event, formatted as
+    JSON in [MozLog](https://wiki.mozilla.org/Firefox/Services/Logging) format.
+
+- `logging.info` (`MERINO_LOGGING__LEVELS`) - Minimum level of logs that should
+  be reported. This should be a number of _entries_ separated by commas (for
+  environment variables) or specified as list (YAML).
+
+  This will be combined with the contents of the `RUST_LOG` environment variable
+  for compatibility. `RUST_LOG` will take precedence over this setting. If the
+  environment variable `MERINO_LOGGING__LEVELS` is specified, all the settings
+  in the YAML file will be ignored.
+
+  Each entry can be one of `ERROR`, `WARN`, `INFO`, `DEBUG`, or `TRACE` (in
+  increasing verbosity), with an optional component that specifies the source of
+  the logs. For example `INFO,merino_web=DEBUG,reqwest=WARN` would set the
+  default log level to INFO, but would lower the level to `DEBUG` for the
+  `merino-web` crate and raise it to `WARN` for the reqwest crate.
+
+### Metrics
+
+Settings for Statsd/Datadog style metrics reporting.
+
+- `metrics.sink_address` (`MERINO_METRICS__SINK_ADDRESS`) - The IP and port to
+  send metrics to over UDP. Defaults to `127.0.0.1:8125`.
+
+- `max_queue_size_kb` (`MERINO_METRICS__MAX_QUEUE_SIZE_KB`) - The maximum size
+  of the buffer that holds events waiting to be sent. If unsent events rise
+  above this, then metrics will be lost. Defaults to 32KB.
+
+### Sentry
+
+Error reporting via Sentry.
+
+- `sentry.mode` (`MERINO_SENTRY__MODE`) - The type of Sentry integration to
+  enable. One of `release`, `debug`, or `disabled`.
+
+If `sentry.mode` is set to `disabled`, no Sentry integration will be activated.
+If it is set to `debug`, the DSN will be set to a testing value recommended by
+Sentry, and extra output will be included in the logs.
+
+If `sentry.mode` is set to `release`, then the following two settings are
+required:
+
+- `sentry.dsn` - Configuration to connect to the Sentry project.
+- `sentry.env` - The environment to report to Sentry. Probably "production",
+  "stage", or "dev".
+
+### Redis
+
+Connection to Redis. This is used by the Redis provider cache below.
+
+- `redis.url` (`MERINO_REDIS__URL`) - The URL to connect Redis at. Example:
+  `redis://127.0.0.1/0`.
+
+### Remote_settings
+
+Connection to Remote Settings. This is used by the Remote Settings suggestion
+provider below.
+
+- `remote_settings.storage_path` (`MERINO_REMOTE_SETTINGS__STORAGE_PATH`) - The
```

Comment:
Killed by #133 in case you want to remove this line now

Reply:
I'll handle that in #133.